static/src/javascripts/projects/common/modules/ui/subscription-banner-template.js

```diff
@@ -39,7 +40,11 @@ const subscriptionBannerTemplate = (
             </div>
         </div>
 
-        <div class="site-message--subscription-banner__sign-in">
+        <div class="site-message--subscription-banner__sign-in ${
+            userLoggedIn
+                ? 'site-message--subscription-banner__sign-in--visibility-hidden'
+                : ''
+        }">
```

Comment:
Are you able to do this because the whole thing is wrapped in backticks? I've not seen this before.

Reply:
exactly right everything in `${}` is just javascript